mite/__main__.py

```diff
@@ -282,6 +284,36 @@ def collector(opts):
     asyncio.get_event_loop().run_until_complete(receiver.run())
 
 
+def generic_receiver(opts):
```

Comment:
Can we put this into its own module in `cli`?  I've been trying to do that with all the top level commands I touch, to reduce the nature of this `__main__` file as a big ball of string...